src/core/Form/Toggle/ToggleButton/ToggeButton.baseStyles.tsx

```diff
@@ -1,24 +1,25 @@
 import { css } from 'styled-components';
-import { withSuomifiTheme, TokensAndTheme } from '../../../theme';
 import { absoluteFocus } from '../../../theme/utils';
+import { disabledCursor } from '../../../../components/utils/css';
 import {
   toggleBaseStyles,
   focusOverrides,
 } from '../ToggleBase/Toggle.baseStyles';
 
-export const baseStyles = withSuomifiTheme(
-  (tokensAndTheme: TokensAndTheme) => css`
-    &.fi-toggle--button > button {
-      ${toggleBaseStyles(tokensAndTheme)}
-      &:focus {
-        outline: 0;
-        & .fi-toggle_icon-container {
-          &:after {
-            ${absoluteFocus}
-            ${focusOverrides}
-          }
+export const baseStyles = css`
+  ${toggleBaseStyles}
+  &.fi-toggle--disabled > button {
+    ${disabledCursor}
```

Comment:
Just remembered that in some other place we removed the use of this util and just told the style right away:

```css
cursor: not-allowed;
```

Maybe not the matter of this task, but creating ticket for backlog to get rid of this util?

Reply:
Was such a small effort that replaced all with `cursor: not-allowed;` and removed the whole tooling. These kind of tools with only one function and no updates in the foreseeable future should be removed. Unnecessary abstractions.